oeps/processes/oep-0055-project-maintainers.rst

```diff
@@ -0,0 +1,309 @@
+the scope of this document.  For example, here we will note that
+maintainers are responsible for ensuring that repositories meet
+project standards, but not the details of those standards.  Those
+standards will be codified in an OEP over time, but will initially be
+defined in the wiki page `requirements for public repositories`_ to
+allow rapid iteration.
+
+The key aspects of the maintainer role fall into the following
+categories
+
+* **Community Stewardship**: Maintainers are representatives of the
+  project in the community and are responsible for contributing to an
+  environment that is welcoming, vibrant, and growing.
+* **Project Management**: Maintainers are responsible for ensuring
+  that critical maintenance is done, but this doesn't mean doing it all
+  themselves. Effective delegation is an important maintainer skill.
+* **Quality Assurance**: Maintainers are responsible for the quality
+  of their components.  Quality will mean meeting defined project
+  standards.  It can also mean improving our standards or improving
+  our processes.
+* **Technical Vision**: Maintainers should have a technical vision for
+  their components. This would include key areas of debt that require
+  payoff, a new pattern that should be adopted to improve the
+  component design, a path to add new features user need, and more.
+* **Continuous Improvement**: Maintainers should be thinking
+  about continuously improving both software and processes.
+
+
+
+
+Change History
+==============
+
+A list of dated sections that describes a brief summary of each revision of the
+OEP.
+
+
+Source Material
+===============
+
+* `What Makes a Great Maintainer of Open Source Projects? <https://www.computer.org/csdl/proceedings-article/icse/2021/029600a982/1sEXoQoeO0E>`_
+* `Collective Code Construction Contract <https://rfc.zeromq.org/spec/42/>`_
+
+Notes
+=====
+
+.. [#] An IDA is an Independently Deployable Application, a server-side service oriented application, typically built upon the Django framework.
+
+.. [#] An MFE is a Micro-Front end, a discreet, client side application written in Javascript, typically using the ReactJS framework.
+
+.. [#] TODO: Add sources for best practices claim.
+
+.. [#] Needless to say, there must be feedback loops between
+       maintainers and operators of the platform and, in some cases, a        single person will perform both roles.
+
+.. [#] In fact, studies of the traits of effective maintainers have highlighted that non-technical aspects of the role are often more highly valued by community members 2021 IEEE/ACM 43rd International Conference on Software Engineering (ICSE) `What Makes a Great Maintainer of Open Source Projects? <https://www.computer.org/csdl/proceedings-article/icse/2021/029600a982/1sEXoQoeO0E>`_
+
+
+
+.. _Django framework: https://www.djangoproject.com/
+
+.. _Core Contributor Program: https://github.com/openedx/open-edx-proposals/pull/275
```

Comment:
https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0054-core-contributors.html

Reply:
Closing to de-clutter.  This has been fixed.